python_modules/libraries/dagster-dbt/dagster_dbt_tests/core/test_asset_checks.py

```diff
@@ -214,8 +215,8 @@ def test_with_asset_checks(
         raise_on_error=False,
     )
 
-    assert not result.get_asset_observation_events()
-    assert result.get_asset_check_evaluations()
+    assert len(result.get_asset_observation_events()) == 1
+    assert len(result.get_asset_check_evaluations()) == 24
```

Comment:
Add comments for why we expect this (e.g. dbt relationship tests, singular test)